packages/common/src/store/notifications/types.ts

```diff
@@ -179,6 +180,36 @@ export type RepostPushNotification = {
   }
 }
 
+export type RepostOfRepostNotification = BaseNotification & {
+  type: NotificationType.RepostOfRepost
+  entityId: ID
+  userIds: ID[]
+  entityType: Entity.Playlist | Entity.Album | Entity.Track
+}
+
+export type RepostOfRepostPushNotification = {
+  blocknumber: number
+  entityId: ID
+  initiator: ID
+  timestamp: string
+  type:
+    | PushNotificationType.RepostAlbum
```

Comment:
should there be a new PushNotificationType for repost of repost or we can reuse repost types here?

Reply:
Yeah good q, in this case i think it might be nice to use the same repost type cause imo they should be in lockstep?

Reply:
i think it may be cleaner to have a separate repost of repost type but i trust whatever you decide! i'm not familiar with this code and what `PushNotificationType` is used for. also what's lockstep?